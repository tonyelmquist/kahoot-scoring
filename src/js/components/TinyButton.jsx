import React, { Component } from "react";
import { AwesomeButton } from "react-awesome-button";

class TinyButton extends React.Component {
  constructor(props) {
    super(props);
  }

  render() {
    return (
      <AwesomeButton
        className="tiny-button"
      >
        {this.props.name}
      </AwesomeButton>
    );
  }
}

export default TinyButton;
