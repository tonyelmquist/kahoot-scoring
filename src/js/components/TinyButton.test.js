import React from 'react';
import enzymeSetup from '../utils/enzymeSetup';
import {shallow} from 'enzyme';
import TinyButton from './TinyButton';


describe('TinyButton tests', () => {

    let wrapper;

    beforeEach(() => {
        wrapper = shallow(<TinyButton />);
    });

    test('renders correctly', () => {
        expect(wrapper).toMatchSnapshot();
    });

});