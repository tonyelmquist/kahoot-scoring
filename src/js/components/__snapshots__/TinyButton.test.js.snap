// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TinyButton tests renders correctly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <TinyButton />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "class",
    "props": Object {
      "action": null,
      "active": false,
      "blocked": false,
      "bubbles": false,
      "children": null,
      "className": "tiny-button",
      "cssModule": null,
      "disabled": false,
      "element": null,
      "href": null,
      "moveEvents": true,
      "placeholder": false,
      "rootElement": "aws-btn",
      "size": null,
      "style": Object {},
      "target": null,
      "title": null,
      "to": null,
      "type": "primary",
      "visible": true,
    },
    "ref": null,
    "rendered": null,
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "action": null,
        "active": false,
        "blocked": false,
        "bubbles": false,
        "children": null,
        "className": "tiny-button",
        "cssModule": null,
        "disabled": false,
        "element": null,
        "href": null,
        "moveEvents": true,
        "placeholder": false,
        "rootElement": "aws-btn",
        "size": null,
        "style": Object {},
        "target": null,
        "title": null,
        "to": null,
        "type": "primary",
        "visible": true,
      },
      "ref": null,
      "rendered": null,
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
